core/src/mindustry/input/InputHandler.java

```diff
@@ -601,6 +617,44 @@ protected void removeSelection(int x1, int y1, int x2, int y2, boolean flush){
         }
     }
 
+    protected void updateArea(int x1, int y1, int x2, int y2, boolean forced){
+        lineRequests.clear();
+
+        NormalizeResult dresult = Placement.normalizeArea(x1, y1, x2, y2, rotation, false, maxLength);
+        for(int x = dresult.x; x <= dresult.x2; x++){
+            for(int y = dresult.y; y <= dresult.y2; y++){
+                Tile tile = world.tilec(x, y);
+                if(tile.block() == Blocks.air || tile.block() instanceof Rock
+                    || !validPlace(x, y, block, tile.rotation())
+                    || !(tile.block() instanceof Conveyor || tile.block() instanceof Conduit || tile.block() instanceof Wall)) continue;
+
+                if((block instanceof ArmoredConveyor || block instanceof ArmoredConduit) && !forced){
+                    if(tile.left() != null){
+                        Tilec left = tile.left();
+                        if(((Autotiler)tile.block()).blends(tile, tile.rotation(), left.tileX(), left.tileY(), left.rotation(), left.block())) continue;
```

Comment:
Somehow worked the first time, then doesn't after though it should after research. Going to remove in next commit